### PR TITLE
fix: delete status-code response

### PR DESCRIPTION
**Pull request by mohammedzeglam-pg**
### Contributor checklist

- [ ] Bug fixes include regression tests
- [ ] Features include unit/acceptance tests

### CODE REVIEW

File: test/acceptance/delete_test.exs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      %{post: post}
+    end
+
+    test "string attributes are rendered properly", %{post: post} do
+      Api
+      |> delete("/posts/#{post.id}", status: 200)
+    end
+
+    @tag :attributes
+    test "private attributes are not rendered in the payload", %{post: post} do
```
Comment (zachdaniel) [inline test/acceptance/delete_test.exs:109]: This test is unnecessary

File: test/acceptance/delete_test.exs [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      |> delete("/posts/#{id}", status: 404)
+      |> assert_has_error(%{
+        "code" => "NotFound",
+        "detail" => "No post record found with `id: #{id}`",
+        "title" => "Entity Not Found"
+      })
+    end
+  end
+
+  @tag :attributes
```
Comment (zachdaniel) [inline test/acceptance/delete_test.exs:91]: No need for the tag. Perhaps `describe "found"`

File: test/acceptance/delete_test.exs [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      post =
+        Post
+        |> Ash.Changeset.for_create(:create, %{name: "foo", profile: %{bio: "Bio"}})
+        |> Ash.Changeset.force_change_attribute(:hidden, "hidden")
+        |> Api.create!()
+
+      %{post: post}
+    end
+
+    test "string attributes are rendered properly", %{post: post} do
```
Comment (zachdaniel) [inline test/acceptance/delete_test.exs:103]: Call this "delete responds with 200"

**Review by zachdaniel — CHANGES_REQUESTED** [review]

File: test/acceptance/delete_test.exs (hunk 3 above)
Comment (mohammedzeglam-pg) [reply test/acceptance/delete_test.exs]: thanks alot

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**